src/storage/DataAccessorBasedStore.ts

```diff
@@ -347,6 +379,49 @@ export class DataAccessorBasedStore implements ResourceStore {
     }
   }
 
+  /**
+   * Write the given metadata resource to the DataAccessor.
+   * @param identifier - Identifier of the resource.
+   * @param representation - Corresponding Representation.
+   *
+   * @returns Identifiers of resources that were possibly modified.
+   */
+  protected async writeMetadata(identifier: ResourceIdentifier, representation: Representation):
+  Promise<ResourceIdentifier[]> {
+    const subjectIdentifier = this.metadataStrategy.getSubjectIdentifier(identifier);
+
+    // Cannot create metadata without a corresponding resource
+    if (!await this.hasResource(subjectIdentifier)) {
+      throw new ConflictHttpError('Metadata resources can not be created directly.');
+    }
+
+    // https://github.com/CommunitySolidServer/CommunitySolidServer/issues/1027#issuecomment-988664970
+    // It must not be possible to create .meta.meta resources
+    if (this.metadataStrategy.isAuxiliaryIdentifier(subjectIdentifier)) {
+      throw new ConflictHttpError(
+        'Not allowed to create metadata resources on a metadata resource.',
+      );
+    }
+
+    const metadata = new RepresentationMetadata(subjectIdentifier);
+    let rdf = representation;
+    if (!representation.isEmpty) {
+      rdf = await this.converter.handleSafe({
+        identifier,
+        representation,
+        preferences: { type: { [INTERNAL_QUADS]: 1 }},
+      });
+    }
+
+    const quads = await arrayifyStream(rdf.data);
+    metadata.addQuads(quads);
+
+    this.removeResponseMetadata(metadata);
+    metadata.removeAll(CONTENT_TYPE_TERM);
```

Comment:
ResponseMetadata is the metadata that is in the ResponseMetadata graph: https://github.com/CommunitySolidServer/CommunitySolidServer/blob/df2926a52ff7ca45a4a2b19871f293b791360c05/src/util/Vocabularies.ts#L159-L160
This was added at some point to prevent PATCH requests writing such metadata back to the resource.